src/domains/sign/io_sign_backoffice_func.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  backoffice_func_settings = {
+    for s in var.io_sign_backoffice_func.app_settings :
+    s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${module.key_vault.name};SecretName=${s.key_vault_secret_name})" : s.value
+  }
+}
+
+module "io_sign_backoffice_func" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v6.2.1"
+
+  name                = format("%s-backoffice-func", local.project)
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+
+  health_check_path = "/health"
+
+  node_version    = "16"
```

Comment:
18 :)

Reply:
we can't use 18 right now, we'll see next month